datagrepper/app.py

```diff
@@ -773,3 +778,14 @@ def internal_error(error):
         status=500,
         mimetype="application/json",
     )
+
+
+def liveness():
+    pass
+
+
+def readiness():
+    try:
+        print("everything looks good")
```

Comment:
It would be more useful to try a database query, for example something like `SELECT 1`. It would ensure that the database connection is actually working.